### PR TITLE
[FLINK-6447] update aws/emr docs

### PR DESCRIPTION
**Pull request by alpinegizmo**
EMR now has explicit Flink support, so there's no need to install Flink by hand unless you need something custom.

### CODE REVIEW

File: docs/setup/aws.md [hunk 1]
```diff
@@ -32,17 +32,23 @@ Amazon Web Services offers cloud computing services on which you can run Flink.
 
 [Amazon Elastic MapReduce](https://aws.amazon.com/elasticmapreduce/) (Amazon EMR) is a web service that makes it easy to  quickly setup a Hadoop cluster. This is the **recommended way** to run Flink on AWS as it takes care of setting up everything.
 
-### Create EMR Cluster
+### Standard EMR Installation
 
-The EMR documentation contains [examples showing how to start an EMR cluster](http://docs.aws.amazon.com/ElasticMapReduce/latest/ManagementGuide/emr-gs-launch-sample-cluster.html). You can follow that guide and install any EMR release. You don't need to install *All Applications* part of the EMR release, but can stick to *Core Hadoop*:
+Flink is a supported application on Amazon EMR. Basically all you have to do is choose Flink as an application, along with whatever
```
Comment (greghogan) [inline docs/setup/aws.md]: How about something like "Flink is a supported application on Amazon EMR. [Amazon's documentation](http://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-flink.html) describes configuring Flink, creating and monitoring a cluster, and working with jobs."?

File: docs/setup/aws.md [hunk 2]
```diff
@@ -32,17 +32,23 @@ Amazon Web Services offers cloud computing services on which you can run Flink.
...
-When creating your cluster, make sure to setup [IAM roles](http://docs.aws.amazon.com/ElasticMapReduce/latest/ManagementGuide/emr-iam-roles.html) allowing you to access your S3 buckets if required.
+The standard installation (above) is easier, but if you need to use a version of Flink that Amazon doesn't support,
+then you can setup a stock EMR cluster and install Flink yourself.
 
-{% top %}
+**Create EMR Cluster**
+
+The EMR documentation contains [examples showing how to start an EMR cluster](http://docs.aws.amazon.com/ElasticMapReduce/latest/ManagementGuide/emr-gs-launch-sample-cluster.html). You can follow that guide and install any EMR release. You don't need to install the *All Applications* part of the EMR release, but can stick to *Core Hadoop*.
+
+When creating your cluster, make sure to setup [IAM roles](http://docs.aws.amazon.com/ElasticMapReduce/latest/ManagementGuide/emr-iam-roles.html) allowing you to access your S3 buckets if required.
```
Comment (greghogan) [inline docs/setup/aws.md]: How about something like "When creating a cluster, access to S3 buckets requires configuration of [IAM roles](http://docs.aws.amazon.com/ElasticMapReduce/latest/ManagementGuide/emr-iam-roles.html)."? And prefix with our "note" warning?

File: docs/setup/aws.md [hunk 3]
```diff
@@ -32,17 +32,23 @@ Amazon Web Services offers cloud computing services on which you can run Flink.
...
 
-The EMR documentation contains [examples showing how to start an EMR cluster](http://docs.aws.amazon.com/ElasticMapReduce/latest/ManagementGuide/emr-gs-launch-sample-cluster.html). You can follow that guide and install any EMR release. You don't need to install *All Applications* part of the EMR release, but can stick to *Core Hadoop*:
+Flink is a supported application on Amazon EMR. Basically all you have to do is choose Flink as an application, along with whatever
+else you need, and configure the instances and roles. [Amazon's documentation](http://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-flink.html) gives all the details.
 
-<img src="{{ site.baseurl }}/fig/flink-on-emr.png" class="img-responsive">
+### Custom EMR Installation
 
-When creating your cluster, make sure to setup [IAM roles](http://docs.aws.amazon.com/ElasticMapReduce/latest/ManagementGuide/emr-iam-roles.html) allowing you to access your S3 buckets if required.
+The standard installation (above) is easier, but if you need to use a version of Flink that Amazon doesn't support,
```
Comment (greghogan) [inline docs/setup/aws.md]: How about something like "Amazon EMR services are regularly updated to new releases but a version of Flink which is not available can be manually installed within a stock EMR cluster."?

**Comment by alpinegizmo** [comment]
@greghogan Thanks for the feedback!

**Comment by StephanEwen** [comment]
+1 to merge this!

**Pull request closed without merging**